lottie/src/main/java/com/airbnb/lottie/model/layer/TextLayer.java

```diff
@@ -459,7 +535,8 @@ private void drawGlyph(Path path, Paint paint, Canvas canvas) {
     canvas.drawPath(path, paint);
   }
 
-  private void drawCharacterFromFont(String character, DocumentData documentData, Canvas canvas) {
+  private void drawCharacterFromFont(String character, DocumentData documentData, Canvas canvas, int indexInDocument, int parentAlpha) {
+    configurePaint(documentData, parentAlpha, indexInDocument);
```

Comment:
This means we need to configure the paint on every character. I haven't looked into the performance implication or further optimization.